modules/ROOT/pages/autocorrect.adoc

```diff
@@ -0,0 +1,44 @@
+= Spelling Autocorrect plugin
+:navtitle: Spelling Autocorrect
+:description: Autocorrect common spelling mistakes.
+:keywords: autocorrect, capitalization, spell, spelling, spellcheck
+:pluginname: Spelling Autocorrect
+:plugincode: autocorrect
+:pluginminimumplan: tiertwo
+
+
+include::partial$misc/admon-premium-plugin.adoc[]
+
+The {pluginname} plugin automatically corrects a specified set of spelling errors, capitalization errors, or both, as they are being typed.
```

Comment:
Is this accurate? Currently it doesn't do both, or are we listing that as a known issue?

Reply:
Also this is missing a major part that needs to be documented... it only works for English.